packages/App/src/onBoardingPages/Paginator.tsx

```diff
@@ -0,0 +1,49 @@
+import React, {FC} from 'react';
+import {View, StyleSheet, Animated} from 'react-native';
+import {OnBoardingInfo} from './OnBoarding';
+
+interface Props {
+  data: Array<OnBoardingInfo>;
```

Comment:
Your `Paginator` component cannot be reused elsewhere as it depends on `OnBoardingInfo` items. It should not: it only depends on the number of items (lengths of the array) and on index of active item